docs/source/user guide/controller configuration.rst

```diff
@@ -28,22 +28,22 @@ whether or not your relay requires the pin to be inverted (for most mechanical r
     you have an actual door sensor connected to your controller. If you set this up incorrectly your controller may think
     the door to your fermentation chamber is always open, and fail to trigger heating/cooling as expected.
 
-
 Temperature Sensors
 ---------------------------
 
-OneWire temperature sensors are configured by device address rather than the pin they are connected to. Connect your
-OneWire sensors to your BrewPi controller, wait a few seconds, and refresh the page. You should see all connected
-sensors listed under either Available Devices or Installed Devices.
+Fermentrack supports the DS18B20 temperature sensors, which use the OneWire protocol. This allows multiple sensors to be connected via a single pin on the controller board. For the ESP8622 board, this is the D6 pin. To ensure stable data transfers, you will also want to include a 4.7K resistor between the voltage source and the data pin. An example of how to gang multiple temperature sensors together, with resistor, looks like this:
 
-To install an available temperature sensor, simply choose the device function (Chamber Temp, Room Temp, Beer Temp,
-etc.) and click "Assign". Your BrewPi controller will configure the device appropriately.
+.. image:: https://www.microcontroller-project.com/uploads/2/2/1/5/22159166/nodemcu-esp8266-with-multiple-ds18b20-sensors_orig.png
 
+Again, for the ESP8622, you will want to connect the data wire to pin D6.
```

Comment:
I would try to avoid referencing any specific hardware outside of pages dedicated to that specific build. Functionality here is referring to the configuration of a (presumably properly wired) controller - but we don’t know what kind of controller the user has connected (Arduino, ESP8266, ESP32, Fuscus, etc.)

Reply:
Not related to this comment, but like I asked above- is D6 hardcoded for the ESP? Would it be different on the other boards? That info can help me improve this sentence.

Reply:
Deleted the sentence you highlighted. Is that sufficient for this?